testkit/testkit.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !codes
```

Comment:
please revert this.

Reply:
Sorry, the goland in go1.17 convert those flags automatically. I will fix it.